<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogXimea</class>
 <widget class="QDialog" name="dialogXimea">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>387</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBoxIntegration">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Integration</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_gain">
        <property name="text">
         <string>Gain</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="SliderWidget" name="sliderWidget_Gain">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string> %</string>
        </property>
        <property name="tracking">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_integration_time">
        <property name="text">
         <string>Integrationtime</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="SliderWidget" name="sliderWidget_integrationtime">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
        <property name="suffix">
         <string> ms</string>
        </property>
        <property name="tracking">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_framerate">
        <property name="text">
         <string>Framerate</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="SliderWidget" name="sliderWidget_framerate">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string> ms</string>
        </property>
        <property name="tracking">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QGroupBox" name="groupBoxSize">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Size</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="lblx0">
          <property name="text">
           <string>x0</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="lblx1">
          <property name="text">
           <string>x1</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lbly0">
          <property name="text">
           <string>y0</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="RangeWidget" name="rangeY">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>999999999</number>
          </property>
          <property name="maximumValue">
           <number>99</number>
          </property>
          <property name="maximumRange">
           <UInt>999999999</UInt>
          </property>
          <property name="prefix">
           <string/>
          </property>
          <property name="suffix">
           <string/>
          </property>
          <property name="tickInterval">
           <number>5</number>
          </property>
          <property name="autoSpinBoxWidth">
           <bool>true</bool>
          </property>
          <property name="tracking">
           <bool>false</bool>
          </property>
          <property name="decimals" stdset="0">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="lbly1">
          <property name="text">
           <string>y1</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="RangeWidget" name="rangeX">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>999999999</number>
          </property>
          <property name="maximumValue">
           <number>99</number>
          </property>
          <property name="maximumRange">
           <UInt>999999999</UInt>
          </property>
          <property name="prefix">
           <string/>
          </property>
          <property name="suffix">
           <string/>
          </property>
          <property name="tickInterval">
           <number>5</number>
          </property>
          <property name="autoSpinBoxWidth">
           <bool>true</bool>
          </property>
          <property name="tracking">
           <bool>false</bool>
          </property>
          <property name="decimals" stdset="0">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_sizeX_2">
          <property name="text">
           <string>x-size</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinSizeX">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="keyboardTracking">
           <bool>false</bool>
          </property>
          <property name="suffix">
           <string> px</string>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
          <property name="value">
           <number>666</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblsizey">
          <property name="text">
           <string>y-size</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinSizeY">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="suffix">
           <string> px</string>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
          <property name="value">
           <number>666</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnFullROI">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>full ROI</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBoxBinning">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Binning und bpp</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="QComboBox" name="combo_bin">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>28</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_binning">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Binning</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_bpp_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>bpp</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QComboBox" name="combo_bpp">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>General Information</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Sensor type:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_sensor">
        <property name="text">
         <string>unknown</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Serial Nr.: </string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_serial">
        <property name="text">
         <string>unknown</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RangeWidget</class>
   <extends>QWidget</extends>
   <header>rangeWidget.h</header>
  </customwidget>
  <customwidget>
   <class>SliderWidget</class>
   <extends>QWidget</extends>
   <header>sliderWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
